wallet/chain/p/context.go

```diff
@@ -45,38 +44,15 @@ func NewContextFromClients(
 		return nil, err
 	}
 
-	// TODO: After Etna is activated, assume the gas price is always non-zero.
-	if dynamicFeeConfig.MinPrice != 0 {
-		_, gasPrice, _, err := chainClient.GetFeeState(ctx)
-		if err != nil {
-			return nil, err
-		}
-
-		return &builder.Context{
-			NetworkID:         networkID,
-			AVAXAssetID:       avaxAssetID,
-			ComplexityWeights: dynamicFeeConfig.Weights,
-			GasPrice:          gasPriceMultiplier * gasPrice,
-		}, nil
-	}
-
-	staticFeeConfig, err := infoClient.GetTxFee(ctx)
+	_, gasPrice, _, err := chainClient.GetFeeState(ctx)
 	if err != nil {
 		return nil, err
 	}
 
 	return &builder.Context{
-		NetworkID:   networkID,
-		AVAXAssetID: avaxAssetID,
-		StaticFeeConfig: fee.StaticConfig{
```

Comment:
Can we remove `StaticFeeConfig` entirely? Perhaps we should add a TODO to the context to remove that field (as I don't think we need it anymore).

Reply:
created this, will tackle in another pr
https://github.com/ava-labs/avalanchego/issues/3601